tests/Functional/Synapse/Connection/WlmContextTest.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Keboola\TableBackendUtils\Functional\Synapse\Connection;
+
+use Doctrine\DBAL\DriverManager;
+use Keboola\TableBackendUtils\Connection\Synapse\SynapseDriver;
+use Tests\Keboola\TableBackendUtils\Functional\Synapse\SynapseBaseCase;
+
+class WlmContextTest extends SynapseBaseCase
+{
+    public function testSynapseWlmContext(): void
+    {
+        $conn = DriverManager::getConnection([
```

Comment:
toto je jen tady pro tento test. Ale nechceme to takto (i za cenu nevyplneneho wlm) pouzit i jinde? At maximum veci bezi pres tento driver? je to sice v SynapseBaseCase, ale v BaseAuthTestCase to neni

Reply:
Přidal sem to všude + sem přidal testSynapseWlmContextNotSet https://github.com/keboola/php-table-backend-utils/pull/81/commits/ee5cf34ddcbe464c6185770ce2444ca2bf149fa2